components/org.wso2.carbon.identity.application.authenticator.fido/src/main/java/org/wso2/carbon/identity/application/authenticator/fido/util/FIDOAuthenticatorConstants.java

```diff
@@ -37,14 +37,29 @@ private FIDOAuthenticatorConstants() {
 
     public static final String FIDO_AUTH = "FidoAuth";
     public static final String FIDO2_AUTH = "Fido2Auth";
+    public static final String FIDO2_ENROL = "Fido2Enrol";
+    public static final String FIDO2_PASSKEY_STATUS = "Fido2PasskeyStatus";
+    public static final String FIDO2_IDENTIFIER_FIRST = "Fido2IdentifierFirst";
     public static final String APP_ID = "AppID";
 
     public static final String URI_LOGIN = "login.do";
     public static final String URI_FIDO_LOGIN = "fido-auth.jsp";
+    public static final String URI_FIDO2_ENROL = "fido2-enrol.jsp";
+    public static final String URI_FIDO2_PASSKEY_STATUS = "fido2-passkey-status.jsp";
+    public static final String URI_FIDO2_IDENTIFIER_FIRST = "fido2-identifierfirst.jsp";
 
     public static final String WEBAUTHN_ENABLED = "FIDO.WebAuthn.Enable";
-    public static final String AUTHENTICATOR_FIDO = "authenticator.Fido";
```

Comment:
Don't we have any usage for this? It's not correct to remove public contacts.

Reply:
Resolved via [59395d7](https://github.com/wso2-extensions/identity-local-auth-fido/pull/110/commits/59395d7a0c1dad27440c38d7ed768bb016868b68)